test/test_utils/sagemaker.py

```diff
@@ -240,27 +241,6 @@ def generate_sagemaker_pytest_cmd(image, sagemaker_test_type):
     )
 
 
-def install_sm_local_dependencies(framework, job_type, image, ec2_conn, ec2_instance_ami):
```

Comment:
I thought this would cause the coverage doc test to fail, but it looks like it is actually failing due to a different issue (seems related to tenacity) fyi

Reply:
Fixed it with [928fc9b](https://github.com/aws/deep-learning-containers/pull/2437/commits/928fc9b91caebbcc57770f99e63065ef90bc6cf4)